.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,77 +6,63 @@ labels: "Type: Bug"
-```python
-def a_func():
-    return 1
+IMPORTANT NOTES
 
-a = 1
-b = a_func()
-print(a + b)
-```
-This is preferred over screen shots since it is searchable and others can copy/paste
-the text to run it.
+1. Replace all example text (contained in "<>") or anywhere specifically commenting to replace the
+   text, leaving any guiding HTML comments in place (formatted like this large block so it won't
+   show up in your Bug Report text.)
+2. Use GH flavored markdown: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax,
+   especially for code snippets, which should look like the following:
+   ```python
+   a = 1
+   b = 2
+   print(a + b)
+   ```
+3. Please be as thorough as possible when describing what went wrong, and what was expected from a
+   correct solution. This might look different depending on the nature of the bug, but more
+   information is always best to ensure you receive the help you need.
 -->
 
 # Add meaningful title here
-<!--
-A clear and concise description of the bug including what happened
-and what you expected to happen.
--->
+<!--The title should clearly define the issue succinctly.-->
+
+<Describe your bug/issue here using as much detail as necessary.>
 
 ## How to reproduce
-<!--
-Describe how another person with no context can recreate this issue.
-It is typically very helpful to reduce the problem as much as possible
-and share a minimum working example. See the note above on including
-code as text rather than screenshots.
--->
+
+<Describe how another person with no context can recreate this issue.>
 
 ## Relevant output
-<!--
-Include any output, plots, or other means of communication here to add context to the problem.
- -->
 
-## GreenHEART version
-<!--
-Share your GreenHEART version and how you installed it. You can print the GreenHEART version from
-a Python REPL or script with these commands:
-```python
-import greenheart
-print(greenheart.__version__)
-```
- -->
+<Include any output, plots, tracebacks, or other means of communication here to add context to the
+problem. All code and tracebacks should be properly markdown formatted.>
 
 ## System Information
 <!-- Add your information here. -->
- - OS: <e.g. Ubuntu 20.04 or macOS 10.12>
- - Python version: <Result of `python --version`>
- - Relevant library versions, from the results of `pip freeze`, for example:
-   - FLORIS
-   - HOPP
-   - matplotlib
-   - NREL-PySAM
-   - numpy
-   - numexpr
-   - orbit-nrel
-   - pandas
-   - scipy
-   - shapely
+- OS: <macOS 12.4>
+  <!-- e.g. Ubuntu 20.04 or macOS 10.12 -->
+- Python version: <3.10.4>
+  <!-- All OS: `python --version`-->
+- GreenHEART version: <0.1.1>
+  <!--
+  Unix: `pip list --format freeze | grep greenheart`
+  Windows: `pip list --format freeze | findstr greenheart`
+  -->
+  - <Installed from source using an editable installation with developer tools: `pip install -e .[develop]`>
```

Comment:
Add a note that if you're on a developer install to include the commit hash?

Reply:
Sure thing, @johnjasa do you know what the below Windows equivalent would be for getting "hash" in this pattern `git@hash#egg=greenheart`?

Reply:
Realizing the update doesn't show up in the thread, but I added this comment for getting the hash:
`Unix: pip freeze | grep greenheart | awk -F"git@" '/git@/{print $2}' | awk -F"#egg" '/#egg/{print $1}'`

Reply:
I'm not sure immediately; maybe this? https://gist.github.com/mamchenkov/930900?permalink_comment_id=3620234#gistcomment-3620234

Reply:
I don't think that's the same because I'm trying to get the information for what pip has installed. This is particularly helpful in the case of accidentally not installing with `-e`. Does the following work in PowerShell, or do Windows people use something different? Or, will Anaconda prompt work with the above command?

```
$name = pip freeze | grep greenheart
$pos1 = $name.IndexOf("@git") + 4
$pos2 = $name.IndexOf("#egg")
$name.Substring($pos1, $pos2 - $pos1)
```

Reply:
I just updated the comment to copy the hash between the substrings of interest, so we don't have to spend a significant amount of time figuring this one out.